src/app/components/TargetPrice/TargetPrice.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import styles from './TargetPrice.module.css';
+
+type TargetPriceProps = {
+  description: string;
+  targetPrice: string;
+};
+
+function TargetPrice({
+  description,
+  targetPrice,
+}: TargetPriceProps): JSX.Element {
+  return (
+    <div className={styles.container}>
```

Comment:
What's the reason for this container? It looks like the CSS class doesn't exist.

Reply:
The original idea was to build the target price as a component, because the props and the target price come from different sources, but I want to display them in one tile. But I have solved it differently and can remove it